Add CUAV V5+ Build Log

```diff
@@ -0,0 +1,176 @@
+# DJI F450 + CUAV V5+ Build
+
+This topic provides full instructions for building the kit and configuring PX4 using *QGroundControl*.
+
+Key information
+
+- **Frame:** DJI F450
+- **Flight controller:** [CUAV V5+]
+- **Assembly time (approx.):** 1.5 hours (1 for frame, 0.5 autopilot installation/configuration)
+
+## Bill of materials
+
+* Power module holybro
```

Comment:
Which one? http://docs.px4.io/master/en/power_module/cuav_hv_pm.html ?

Reply:
Power Module used is PM02 for the X500 Buildlog